beakerx/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "beakerx",
-  "version": "0.6.3-widgets7b",
+  "version": "0.7.1",
```

Comment:
The version no. is changed due to issues with the `0.6.2-widgets7b` version that can't be used in jupyterlab.
This version should be used to local tests only.